src/analyser.cpp

```diff
@@ -282,7 +282,7 @@ bool AnalyserInternalEquation::check(size_t &equationOrder, size_t &stateIndex,
         if (variable->mType == AnalyserInternalVariable::Type::UNKNOWN) {
```

Comment:
I'm not sure what changes in this file have to do with this PR?  Also the test_utils.cpp ones?

Reply:
For this file, it has to do with ClangFormat. In CF 11, they "properly" indent the ternary operator.

Regarding `tests/test_utils.cpp`, it's to prevent a warning on Windows.